assets/styles/components/texts.scss

```diff
@@ -80,6 +82,8 @@ html[data-theme='dark'] {
 
   p {
     color: get-dark-color('text-color');
+    text-align: justify;
+    text-justify: inter-word;
```

Comment:
You don't need to repeat these lines. Only add CSS that should be changed for dark mode.

Reply:
Ok, thank you for this advice. Now it's fixed.